src/Appwrite/Utopia/Response/Model/AuthProvider.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class AuthProvider extends Model
```

Comment:
@fanatic75 Let's add a `Providers` directory for both auth and messaging and use that as the namespace for both. Same for the config files, just lowercase.